modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -11,6 +11,7 @@ import DOMUtils from './api/dom/DOMUtils';
 import { EventUtilsEvent } from './api/dom/EventUtils';
 import EditorSelection from './api/dom/Selection';
 import Editor from './api/Editor';
+import { VK } from './api/PublicApi';
```

Comment:
This is the wrong import and as such it's causing a circular dependency. It should be this (not a suggestion as it needs to be moved to the right place):
```
import VK from '../api/util/VK';
```